src/ceph-volume/ceph_volume/devices/raw/activate.py

```diff
@@ -107,7 +113,8 @@ def main(self):
             description=sub_command_help,
         )
         parser.add_argument(
-            '--device',
```

Comment:
Ah I see, this is going to break Rook's master branch. Are these changes planned to be backported?